### DISCUSSION TITLE
7th September 2022 - MSF Delegate Sync -

### DISCUSSION THREAD

**Discussion by futuristudios** [omigroup/MSF-Delegates#16] [General]
MSF Delegate Sync -
--
_Wednesday 7th September 2022. (1500hrs UTC)_

### agenda thus far:
- 1500 utc: @funksoup work on the [miro]   (https://github.com/omigroup/MSF-Delegates/discussions/14#discussioncomment-3519291) Dive back into [MSF brainstorm/scope of work miro board](https://miro.com/app/board/uXjVPe7tenM=/) to better define mission/scope of work and relate back to kanban items.
- 1530 utc: @pcsmgnt touch base with @mrmetaverse to cover github `[workflows]`(https://github.com/omigroup/MSF-Delegates/discussions/14#discussioncomment-3519387) Jesse Alton Github Repo Tips and Tricks
- ...
- draft the agenda for next week [^1]

---
[^1]: _next meeting_: Wednesday 14th September 2022. (1500hrs UTC) [subscribe to OMI calendar](https://calendar.google.com/calendar/ical/c_0dpugf9f83eq4qekmb6omqbjmg%40group.calendar.google.com/public/basic.ics)

**Comment by funksoup** [comment]
[09/07/22 Metaverse Standards Forum: Oversight call - slides & notes](https://github.com/omigroup/MSF-Delegates/discussions/17)

**Comment by funksoup** [comment]
Discussion about participation - supporter vs active contributor. If someone wants to be in more of a 'supporter' role that is more passive, perhaps there is a process/role they can take in which they define their ability to support, i.e., I will be a 'passive' watcher of this issue/group -- while in other subgroups, they may take a more 'active contributor' role.

**Reply by funksoup** [reply]
Via @pcsmgnt - is there a better/more precise word than 'supporter'?

**Reply by funksoup** [reply]
Via @humbletim - what does this mean for how SDO's participate? (@pcsmgnt has also asked this question)

**Comment by funksoup** [comment]
@mrmetaverse arrived and gave an overview of github project kanban functionality:
- added "Review" column before "Done"
- Suggested format: "As a [role], I need [task] so that [reason for task]"
example: "As a MSF delegate, I need to have a bot installed so that I am reminded of our upcoming events"
- Can create task lists by using markdown
example:
- [x] task 1 
- [ ] task 2 
- [ ] task 3
- Can convert a comment into a task (hit 3 dots next to comment)
- His 'favorite thing of all' is milestones ("Reference in new issue")
- Can set dependencies by using sub-tasks
- Can keep track of dates by using MILESTONES
